docs/product-installation-guide/src/main/asciidoc/assembly_installing-on-openshift.adoc

```diff
@@ -1,9 +1,9 @@
-[id='openshift_bxms_con']
+[id='openshift_ba-dm-con']
```

Comment:
Should be called 'openshift-ba-dm-assembly'. No underscore and not con. I'm assuming the underscore is to differentiate the assembly from the actual con file, but naming it correctly as an assembly negates that.